conf/brs.properties.h2

```diff
@@ -6,6 +6,6 @@ API.allowed = *
 
 # Database connection JDBC url
 # Append ;AUTO_SERVER=TRUE to enable automatic mixed mode access.
-DB.Url=jdbc:h2:file:/db/burst;DB_CLOSE_ON_EXIT=FALSE
+DB.Url=jdbc:h2:file:./burst_db/burst.h2;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
This would be a breaking change i guess, and a relative path is also discouraged for the docker build, where you'd need to modify the volume definition as well in Dockerfile.h2

Reply:
Thanks for the feedback. I was not sure what was the intent of this file (was just trying to keep it consistent with brs-default). I'll revert this.